[GraphQL] Only exposes the mutations if any

sroze opened the pull request:
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | ø
| License       | MIT
| Doc PR        | ø

Otherwise, GraphiQL gives this lovely message:
> Mutation fields must be an object with field names as keys or a function which returns [...]

dunglas commented on src/GraphQl/Type/SchemaBuilder.php:
```diff
@@ -92,16 +92,21 @@ public function getSchema(): Schema
-            'mutation' => new ObjectType([
+        ];
+
+        if (\count($mutationFields) > 0) {
```
`if ($mutationFields)`?

alanpoulain commented:
Maybe you could add a unit test or a Behat one (by using introspection)?

sroze commented:
Rebased and changed check with a simple `if`.

@alanpoulain based on the complexity of the `SchemaBuilderTest`, I don't think it's worth it. Also, I'll follow up with a fair amount of pull-requests regarding the messenger integration in the next few weeks that should cover this use-case :)

dunglas commented:
Thanks @sroze
